animations/hero-animations/index.md

```diff
@@ -0,0 +1,670 @@
+image from a circular shape to a square shape during flight.
+
+* TOC Placeholder
+{:toc}
+
+<aside class="alert alert-info" markdown="1">
+**New to Flutter?**
+You should first understand how to create a layout using Flutter’s
+widgets.  For more information, see [Building Layouts in
+Flutter](/tutorials/layout/).
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Terminology:**
+A [_Route_](/routing-and-navigation/) describes a page or screen
+in a Flutter app.
+</aside>
+
+You can create this animation in Flutter with Hero objects.
+When a new route appears, a hero animates from the current route to
+the new one. Typically, heroes are small parts of the UI, like
+images, that both routes have in common. From the user's
+perspective the hero "flies" between the routes.
+This guide shows how to create the following hero animations:
+
+<ul>
+<li markdown="1">Standard hero animations<br>
+A _standard hero animation_ flies the hero from one route to a new
+route, usually landing at a different location and with a
+different size.
+
+The following video (recorded at slow speed) shows a typical example.
+Tapping the flippers in the center of the route flies them to the
+upper left corner of a new, blue route, at a smaller size.
+Tapping the flippers in the blue route
+(or using the device's back-to-previous-route gesture)
+flies the flippers back to the original route.
+
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/uF7_JfhBsww" frameborder="0" allowfullscreen></iframe></center>
+</li>&nbsp;
+
+<li markdown="1">Radial hero animations<br>
+In _radial hero animation_, as the hero flies between routes
+its shape appears to change from circular to rectangular.
+
+The following video (recorded at slow speed),
+shows an example of a radial hero animation. At the start,
+a row of three images appears at the bottom of the route.
+Tapping any of the circles flies that image to a new route
+that displays it with a square shape.
+Tapping the square image flies the hero back to
+the original route, displayed with a circular shape.
+
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/V735TEHW5UI" frameborder="0" allowfullscreen></iframe></center>
+</li>
+</ul>
+
+Before moving to the sections specific to
+[standard](#standard-hero-animations)
+or [radial](#radial-hero-animations) hero animations,
+read [basic structure of a hero animation](#basic-structure)
+to learn how to structure hero animation code,
+and [behind the scenes](#behind-the-scenes) to understand
+how Flutter performs a hero animation.
+
+<a name="basic-structure"></a>
+## Basic structure of a hero animation
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* Use two hero widgets with matching tags to implement the transition.
```

Comment:
two hero widgets in different routes but with matching tags

Saying all that might help the reader understand the connection to the following point about the Navigator and its stack of routes